openmetadata-ui/src/main/resources/ui/src/components/AddIngestion/AddIngestion.component.tsx

```diff
@@ -394,6 +394,8 @@ const AddIngestion = ({
       }
     } else if (showDBTConfig && activeIngestionStep === 2) {
       nextStep = activeIngestionStep + 2;
+    } else if (showDBTConfig && activeIngestionStep === 2) {
+      nextStep = activeIngestionStep + 2;
```

Comment:
duplicated with the previous else-if condition?

Reply:
Done

Reply:
thanks folks